src/client/client-wrapper.ts

```diff
@@ -87,4 +90,13 @@ export class ClientWrapper {
 
     return result;
   }
+
+  public async evaluateUrls(urls: string[]) {
```

Comment:
This is great!

If I recall correctly, `Promise.all` will reject immediately on first rejection, which means it would only reject/throw the first broken link, but ignore any subsequent broken links (if there are any).

It would be great to capture all broken links, rather than forcing the user to find a broken link, fix it, run the test again, find the next broken link, etc.

Reply:
Yes. It works like that. Now I am finding a way to execute `requests` all in one go but still manage to get all the failing urls. Maybe a package can help us to achieve this?